src/apps/content-editor/src/app/views/ItemEdit/components/Header/LanguageSelector/LanguageSelector.js

```diff
@@ -59,7 +50,7 @@ export const LanguageSelector = connect((state, props) => {
           className={cx(styles.LanguageSelector, props.className)}
           value={props.selectedLang}
           onSelect={handleSelect}
-          loading={loading}
+          loading={false}
```

Comment:
Should this prop just be removed? I assume we no longer need it since language loading has been hoisted to the app boot?